Security/troubleshooting/.NET/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Syncfusion.Pdf.Parsing;
+using Syncfusion.Pdf.Security;
+using System.Security.Cryptography;
+
+// Open a FileStream for reading the PDF file.
+FileStream fileStream = new FileStream("OwnerPasswordOnly.pdf", FileMode.Open, FileAccess.Read);
+// Load the PDF document from the FileStream with the specified owner password.
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument(fileStream, "12345");
+// Access the security settings of the loaded PDF document.
+PdfSecurity security = loadedDocument.Security;
+// Disable incremental update for the file structure.
+loadedDocument.FileStructure.IncrementalUpdate = false;
+// Set specific permissions for a PDF document.
+loadedDocument.Security.Permissions = PdfPermissionsFlags.EditAnnotations | PdfPermissionsFlags.AssembleDocument;
```

Comment:
If we are going to remove the security, hence can we set the Permissions as default instead of setting this permissions.